packages/react-instantsearch-hooks-dom/src/widgets/__tests__/ClearRefinements.test.tsx

```diff
@@ -0,0 +1,550 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import {
+  InstantSearch,
+  useCurrentRefinements,
+  useRefinementList,
+} from 'react-instantsearch-hooks';
+
+import { createSearchClient } from '../../../../../test/mock';
+import { wait } from '../../../../../test/utils';
+import { ClearRefinements } from '../ClearRefinements';
+
+import type {
+  InstantSearchProps,
+  UseRefinementListProps,
+  UseCurrentRefinementsProps,
+} from 'react-instantsearch-hooks';
+
+const searchClient = createSearchClient();
+
+type SearchProviderProps = {
+  children: React.ReactNode;
+} & Partial<InstantSearchProps>;
+
+const consoleError = jest.spyOn(console, 'error');
+
+beforeAll(() => {
+  consoleError.mockImplementation(() => {});
+});
+
+afterAll(() => {
+  consoleError.mockRestore();
+});
```

Comment:
React is complaining that changes aren't wrapped in an `act` because InstantSearch first renders with default values, then updates the UI. We're using this workaround in the `DynamicWidgets` component as well (c.f. internal conversation from yesterday on React tests).

Reply:
I noticed these warnings now as well, and I think it might be better to leave them for now and find out why they exist, if we can avoid it somehow (likely related to useEffect+widget mount on render) than ignoring them now already? maybe https://twitter.com/acdlite/status/1496175720971874308 is related?

Reply:
Yes it makes sense. I'll remove this for now since they "only" clutter the output but don't fail tests and we can take global action for the whole suite in the end.